src/features/abi-methods/components/abi-value.tsx

```diff
@@ -14,7 +15,21 @@ export function AbiValue({ abiValue }: Props) {
     return <AbiArrayValue array={abiValue} />
   }
   if (abiValue.type === AbiType.String) {
-    return <span>{`"${abiValue.value}"`}</span>
+    return <span className="text-abi-string">{`"${abiValue.value}"`}</span>
+  }
+  if (abiValue.type === AbiType.Address) {
```

Comment:
It looks like this doesn't support other reference types such as Account, Asset and Transaction

Reply:
my bad, they were handled in `DecodedAbiMethodArguments`, ignore this comment